bootstrap/src/main/java/io/airlift/bootstrap/ConfigurationDidYouMean.java

```diff
@@ -0,0 +1,56 @@
+package io.airlift.bootstrap;
+
+import com.google.common.collect.ImmutableList;
+import me.xdrop.fuzzywuzzy.FuzzySearch;
+
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.Set;
+
+import static com.google.common.base.Verify.verify;
+import static java.lang.Math.min;
+import static java.util.Comparator.comparingInt;
+import static java.util.Objects.requireNonNull;
+
+public class ConfigurationDidYouMean
```

Comment:
Make it package-protected

Reply:
This class needs a better name. Call it "FuzzyMatcher", as that is the essence of what it does.

Reply:
done